opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -188,11 +188,13 @@ message Metric {
 // "current value" for every data point. It should be used for an "unknown"
 // aggregation.
 //
-// A Gauge does not support different aggregation temporalities. Given the
-// aggregation is unknown, points cannot be combined using the same
-// aggregation, regardless of aggregation temporalities. Therefore,
-// AggregationTemporality is not included. Consequently, this also means
-// "StartTimeUnixNano" is ignored for all data points.
+// A Gauge does not support different aggregation temporalities.
+// However, its "StartTimeUnixNano" may follow the semantics of a temporality
+// selection. E.g. "StartTimeUnixNano" may be the time of last collection when
```

Comment:
There is no temporality for Gauge, what does `temporality is delta` mean in this text?

Reply:
This is meant to draft out how we might change the proto description text to accommodate this proposal to add a synchronous gauge instrument: https://github.com/open-telemetry/opentelemetry-specification/pull/3540

The current iteration of the synchronous gauge proposal allows aggregation temporality selection to influence the start time of gauges.